lib/colorls/core.rb

```diff
@@ -152,8 +154,12 @@ def init_git_status(show_git)
     # how much characters an item occupies besides its name
     CHARS_PER_ITEM = 12
 
+    def additional_chars
+      CHARS_PER_ITEM + (@show_git ? 4 : 0) + (@show_inode ? 10 : 0)
+    end
```

Comment:
Since this method is called for every entry (possibly twice) and always returns the same value, it would make sense to make this an instance variable instead which is initialized in the constructor.